### ISSUE TITLE
webpack build : javascript error from util-browser module is stoping marko ui to render further.

### ISSUE THREAD

**Issue by prasanthkv** [marko-js/marko#1446]
### Marko Version: 4.18.22

```
    "@marko/webpack": "^2.0.0",
    "marko": "^4.18.22",
    "marko-widgets": "^7.0.1",
    "css-loader": "^3.2.0",
    "file-loader": "^4",
    "style-loader": "^1.0.0",
    "less": "^3.10.3",
    "less-loader": "^5",
    "monaco-editor-webpack-plugin": "^1.7.0",
    "webpack": "^4.41.2"
```

### Details
While running marko application post webpack bundling we are getting `Cannot read property 'get' of undefined` exception while updating state.
The same code works as expected when severed by marko. 

### Expected Behavior
State change must be propagated as expected and values must be updated.. 

### Actual Behavior
Error on console and further javascript execution is blocked

### Possible Fix

N/A

<details><summary>Additional Info</summary>

### Your Environment

- Environment name and version (e.g. Chrome 39, node.js 5.4): chrome / firefox
- Operating System and version (desktop or mobile): macbook pro
- Link to your project: https://github.com/prasanthkv/marko-serve-example

### Steps to Reproduce

1. fork the above repository 
2. `npm install`
3. `webpack`
4. `open demo.html`
5. click on the spinner 

### Stack Trace

```
Uncaught TypeError: Cannot read property 'get' of undefined
    at getMarkoPropsFromEl (util-browser.js:134)
    at getEventFromEl (event-delegation.js:12)
    at HTMLBodyElement.body.addEventListener.listeners.<computed> (event-delegation.js:110)
```

</details>

**Comment by mlrawlings** [comment]
Fixed in `marko@4.18.24`

**Comment by prasanthkv** [comment]
thanks a ton !!
